<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>piGameCreator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/RES/pi16x16.png</normaloff>:/resources/RES/pi16x16.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="WorkSpaceTabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionCreate_project"/>
    <addaction name="actionOpen_project"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="action_Pack_project"/>
    <addaction name="actionSave_a_ll"/>
    <addaction name="separator"/>
    <addaction name="actionSet_up_collaboration"/>
    <addaction name="action_Preferences"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionPiGameCreator_help"/>
    <addaction name="actionAPI_documentation"/>
    <addaction name="separator"/>
    <addaction name="actionYour_first_game"/>
    <addaction name="actionTutorials"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_updates"/>
    <addaction name="actionLicensing"/>
    <addaction name="separator"/>
    <addaction name="actionVisit_piGameCreator_website"/>
    <addaction name="actionAbout_piGameCreator"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionClose_active_tab"/>
    <addaction name="actionClose_all_tabs"/>
    <addaction name="separator"/>
    <addaction name="actionSkins"/>
    <addaction name="separator"/>
    <addaction name="actionToolbars"/>
    <addaction name="separator"/>
    <addaction name="actionShowLog"/>
    <addaction name="actionShow_Main_page"/>
   </widget>
   <widget class="QMenu" name="menuResources">
    <property name="title">
     <string>Resources</string>
    </property>
    <addaction name="actionCreate_image"/>
    <addaction name="actionCreate_sound"/>
    <addaction name="actionCreate_model"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_class"/>
    <addaction name="actionAdd_global_function"/>
    <addaction name="actionDefine_grid"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_GUI_layer"/>
    <addaction name="actionCreate_Font"/>
    <addaction name="actionCreate_vector_path"/>
    <addaction name="actionCreate_Shader"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_2D_scene"/>
    <addaction name="actionCreate_3D_scene"/>
    <addaction name="separator"/>
    <addaction name="actionImport_resources_from_project"/>
    <addaction name="actionClone_resource"/>
    <addaction name="actionExport_resource"/>
    <addaction name="actionExport_some_resources"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_application_settings"/>
   </widget>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="action_Run"/>
    <addaction name="action_Debug"/>
    <addaction name="actionBuild"/>
    <addaction name="action_Parameters"/>
    <addaction name="actionTarget_options"/>
    <addaction name="separator"/>
    <addaction name="action_Targets"/>
    <addaction name="action_Project_options"/>
   </widget>
   <widget class="QMenu" name="menuDeveloper">
    <property name="title">
     <string>&amp;Developer</string>
    </property>
    <addaction name="actionGameScript_tester"/>
    <addaction name="actionSkin_editor"/>
    <addaction name="actionClass_editor"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuResources"/>
   <addaction name="menu_Project"/>
   <addaction name="menuWindow"/>
   <addaction name="menu_Help"/>
   <addaction name="menuDeveloper"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="projectToolbar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Standard</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate_project"/>
   <addaction name="actionOpen_project"/>
   <addaction name="action_Save"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Project tree</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::ActionMask</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="indentation">
        <number>16</number>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="resourceToolbar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Add resources</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate_image"/>
   <addaction name="actionCreate_sound"/>
   <addaction name="actionCreate_GUI_layer"/>
   <addaction name="actionCreate_Font"/>
   <addaction name="actionAdd_class"/>
   <addaction name="separator"/>
   <addaction name="actionCreate_2D_scene"/>
  </widget>
  <widget class="QToolBar" name="runToolbar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Debug and Run</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Run"/>
   <addaction name="action_Debug"/>
   <addaction name="separator"/>
   <addaction name="action_Targets"/>
   <addaction name="separator"/>
   <addaction name="actionBuild"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/door_in.png</normaloff>:/resources/RES/ffficons/door_in.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/folder.png</normaloff>:/resources/RES/ffficons/folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCreate_project">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/page_white.png</normaloff>:/resources/RES/ffficons/page_white.png</iconset>
   </property>
   <property name="text">
    <string>Create &amp;new project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPiGameCreator_help">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/help.png</normaloff>:/resources/RES/ffficons/help.png</iconset>
   </property>
   <property name="text">
    <string>piGameCreator help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionYour_first_game">
   <property name="text">
    <string>Your first game</string>
   </property>
  </action>
  <action name="actionTutorials">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tutorials</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="actionAbout_piGameCreator">
   <property name="text">
    <string>About piGameCreator</string>
   </property>
  </action>
  <action name="actionVisit_piGameCreator_website">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/world_link.png</normaloff>:/resources/RES/ffficons/world_link.png</iconset>
   </property>
   <property name="text">
    <string>Visit piGameCreator website</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>Check for updates...</string>
   </property>
  </action>
  <action name="actionAPI_documentation">
   <property name="text">
    <string>Help for developers</string>
   </property>
  </action>
  <action name="actionLicensing">
   <property name="text">
    <string>Licensing...</string>
   </property>
  </action>
  <action name="actionShowLog">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/log.png</normaloff>:/resources/RES/log.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionTEST">
   <property name="text">
    <string>TEST</string>
   </property>
  </action>
  <action name="actionClose_active_tab">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/cross.png</normaloff>:/resources/RES/ffficons/cross.png</iconset>
   </property>
   <property name="text">
    <string>Close active &amp;tab</string>
   </property>
   <property name="toolTip">
    <string>Closes the active tag</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionClose_all_tabs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/cross.png</normaloff>:/resources/RES/ffficons/cross.png</iconset>
   </property>
   <property name="text">
    <string>Close &amp;all tabs</string>
   </property>
  </action>
  <action name="actionImport_resources_from_project">
   <property name="text">
    <string>Import resources...</string>
   </property>
  </action>
  <action name="actionCreate_image">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/image_add.png</normaloff>:/resources/RES/ffficons/image_add.png</iconset>
   </property>
   <property name="text">
    <string>Create Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
  </action>
  <action name="actionCreate_sound">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/sound_add.png</normaloff>:/resources/RES/ffficons/sound_add.png</iconset>
   </property>
   <property name="text">
    <string>Create Sound</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+U</string>
   </property>
  </action>
  <action name="actionCreate_GUI_layer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/textfield_add.png</normaloff>:/resources/RES/ffficons/textfield_add.png</iconset>
   </property>
   <property name="text">
    <string>Create GUI layer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+G</string>
   </property>
  </action>
  <action name="actionCreate_2D_scene">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/application_add.png</normaloff>:/resources/RES/ffficons/application_add.png</iconset>
   </property>
   <property name="text">
    <string>Create Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionCreate_3D_scene">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/world_add.png</normaloff>:/resources/RES/ffficons/world_add.png</iconset>
   </property>
   <property name="text">
    <string>Create 3D Scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+3</string>
   </property>
  </action>
  <action name="actionCreate_grid_based_2D_scene">
   <property name="text">
    <string>Create grid-based 2D scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+2</string>
   </property>
  </action>
  <action name="actionClone_resource">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/page_copy.png</normaloff>:/resources/RES/ffficons/page_copy.png</iconset>
   </property>
   <property name="text">
    <string>Clone resource...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionExport_resource">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/disk.png</normaloff>:/resources/RES/ffficons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Export resource...</string>
   </property>
  </action>
  <action name="actionAdd_class">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/brick_add.png</normaloff>:/resources/RES/ffficons/brick_add.png</iconset>
   </property>
   <property name="text">
    <string>Create Class</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionAdd_global_function">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/script_add.png</normaloff>:/resources/RES/ffficons/script_add.png</iconset>
   </property>
   <property name="text">
    <string>Add global function</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+U</string>
   </property>
  </action>
  <action name="actionCreate_model">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/car_add.png</normaloff>:/resources/RES/ffficons/car_add.png</iconset>
   </property>
   <property name="text">
    <string>Create Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+M</string>
   </property>
  </action>
  <action name="actionExport_some_resources">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/disk_multiple.png</normaloff>:/resources/RES/ffficons/disk_multiple.png</iconset>
   </property>
   <property name="text">
    <string>Export some resources...</string>
   </property>
  </action>
  <action name="actionCreate_class_template">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/brick_go.png</normaloff>:/resources/RES/ffficons/brick_go.png</iconset>
   </property>
   <property name="text">
    <string>Create Class template...</string>
   </property>
  </action>
  <action name="actionDefine_grid">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/color_swatch.png</normaloff>:/resources/RES/ffficons/color_swatch.png</iconset>
   </property>
   <property name="text">
    <string>Define grid</string>
   </property>
  </action>
  <action name="actionEdit_application_settings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/cog_edit.png</normaloff>:/resources/RES/ffficons/cog_edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit application settings</string>
   </property>
  </action>
  <action name="actionCreate_Font">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/font_add.png</normaloff>:/resources/RES/ffficons/font_add.png</iconset>
   </property>
   <property name="text">
    <string>Create Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="actionShow_Main_page">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/application_home.png</normaloff>:/resources/RES/ffficons/application_home.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Main page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/disk.png</normaloff>:/resources/RES/ffficons/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save project &amp;as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave_a_ll">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/disk_multiple.png</normaloff>:/resources/RES/ffficons/disk_multiple.png</iconset>
   </property>
   <property name="text">
    <string>Save a&amp;ll projects</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/tango/actions/media-playback-start.png</normaloff>:/resources/RES/tango/actions/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_Debug">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/bug_delete.png</normaloff>:/resources/RES/ffficons/bug_delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Debug...</string>
   </property>
  </action>
  <action name="action_Parameters">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/application_osx_terminal.png</normaloff>:/resources/RES/ffficons/application_osx_terminal.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Parameters...</string>
   </property>
  </action>
  <action name="action_Targets">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/computer.png</normaloff>:/resources/RES/ffficons/computer.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Targets...</string>
   </property>
  </action>
  <action name="action_Project_options">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/cog.png</normaloff>:/resources/RES/ffficons/cog.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Project options...</string>
   </property>
  </action>
  <action name="actionCreate_vector_path">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/chart_line_add.png</normaloff>:/resources/RES/ffficons/chart_line_add.png</iconset>
   </property>
   <property name="text">
    <string>Create &amp;vector path</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/compile.png</normaloff>:/resources/RES/compile.png</iconset>
   </property>
   <property name="text">
    <string>Build...</string>
   </property>
  </action>
  <action name="actionCreate_Shader">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/contrast_high.png</normaloff>:/resources/RES/ffficons/contrast_high.png</iconset>
   </property>
   <property name="text">
    <string>Create Shader</string>
   </property>
  </action>
  <action name="actionTarget_options">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/computer_edit.png</normaloff>:/resources/RES/ffficons/computer_edit.png</iconset>
   </property>
   <property name="text">
    <string>Target options...</string>
   </property>
  </action>
  <action name="actionToolbars">
   <property name="text">
    <string>Toolbars</string>
   </property>
  </action>
  <action name="action_Pack_project">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/package_go.png</normaloff>:/resources/RES/ffficons/package_go.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pack project...</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/cog.png</normaloff>:/resources/RES/ffficons/cog.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionSet_up_collaboration">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/connect.png</normaloff>:/resources/RES/ffficons/connect.png</iconset>
   </property>
   <property name="text">
    <string>Set up &amp;collaboration...</string>
   </property>
  </action>
  <action name="actionSkins">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/RES/ffficons/folder_palette.png</normaloff>:/resources/RES/ffficons/folder_palette.png</iconset>
   </property>
   <property name="text">
    <string>Skins</string>
   </property>
  </action>
  <action name="actionGameScript_tester">
   <property name="text">
    <string>GameScript tester</string>
   </property>
  </action>
  <action name="actionSkin_editor">
   <property name="text">
    <string>Skin editor</string>
   </property>
  </action>
  <action name="actionClass_editor">
   <property name="text">
    <string>Framework editor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
